File: Coding Exercises/TypeScript/Day11/toggleCase.ts
function toggleCase(s: string): string {
  let toggledString = "";
  for (let i = 0; i < s.length; i++) {
      const char = s.charAt(i);

      if (char.toUpperCase() === char) {
          toggledString += char.toLowerCase();
      } else if (char.toLowerCase() === char) {
          toggledString += char.toUpperCase();
      } else {
          toggledString += char;
      }
  }
  return toggledString;
}

File: Coding Exercises/Python/Day11/toggle_case.py
def toggle_case(s):
    toggled_string = ""

    for char in s:
        if char.isupper():
            toggled_string += char.lower()
        elif char.islower():
            toggled_string += char.upper()
        else:
            toggled_string += char

    return toggled_string
